tests20/python_client/testcases/test_collection.py

```diff
@@ -34,20 +34,26 @@ def setup_method(self):
 
     @pytest.fixture(
         scope="function",
-        params=get_invalid_strs
+        params=ct.get_invalid_strs
     )
     def get_invalid_string(self, request):
         yield request.param
 
     @pytest.fixture(
         scope="function",
-        params=get_invalid_strs
+        params=ct.get_invalid_strs
     )
-    def get_invalid_schema(self, request):
+    def get_invalid_schema_type(self, request):
         if request.param is None:
             pytest.skip("None schema is valid")
         yield request.param
 
+    def _create_default_collection(self):
+        c_name = cf.gen_unique_str(prefix)
+        collection, _ = self.collection.collection_init(c_name, schema=default_schema)
+        assert_default_collection(collection, c_name)
```

Comment:
can we merge this _create_default_collection() with apiReq._collection() into one?

Reply:
ok